### ISSUE TITLE
bug: URIError: URI malformed when the language of response in the template

### ISSUE THREAD

**Issue by numlock1052** [janhq/jan#2064]
I just asked Jan with this request : 

```
Donne moi un code Django d'une vue admin avec un model de type contact qui possède deux champ, first_name et last_name et qui inclu un template html django et la possibilité de choisir les colonnes visibles
```

I configured the template with these settings :

```
[INST] answer in french {prompt} [/INST]
```

I used the Mistral model downloaded from the Hub and if the Stream setting is enabled the client raises this error :

![image](https://github.com/janhq/jan/assets/10225601/5111866a-c9a7-4496-a7cf-a32b0ba880d9)

If the Stream is enabled :

![image](https://github.com/janhq/jan/assets/10225601/33e08136-814c-49d7-947c-a7fd5a94726b)

If the Stream is disabled :

![image](https://github.com/janhq/jan/assets/10225601/cdbdc33e-1e6f-41f0-9946-148ebbe04575)

I did the test in the German language, we have another error in the terminal but it's finishing the response, but the content is not the same, so it's not relevant...

![image](https://github.com/janhq/jan/assets/10225601/602a9a7c-873a-4989-82d2-46aa3330e7a2)

```
URIError: URI malformed
Please report this to https://github.com/markedjs/marked.
    at decodeURIComponent (<anonymous>)
    at _Renderer.code (page-e1f7c19f030c0381.js:3467:1130)
    at renderer.<computed> [as code] (178-3df7d54a6164e6f3.js:76460:48)
    at _Parser.parse (178-3df7d54a6164e6f3.js:76126:42)
    at parse (178-3df7d54a6164e6f3.js:76086:23)
    at Marked. parse (178-3df7d54a6164e6f3.js:76582:28)
    at SimpleTextMessage (page-e1f7c19f030c0381.js:3471:31)
    at Mf (618f8807-c0370e0f50991ad5.js:115:143)
    at Bh (618f8807-c0370e0f50991ad5.js:164:172)
    at Ah (618f8807-c0370e0f50991ad5.js:161:204)
```

**Comment by louis-jan** [comment]
cc: @urmaur, regarding MarkedJS.

**Comment by Van-QA** [comment]
hi @numlock1052, we resolved the issue and tested as of Jan v0.4.9-343

Would you be able to try it out? feel free to reopen the issue if the issue persist 🙏
